test/sql/new_feature_test.cpp

```diff
@@ -0,0 +1,198 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Can you separate the tests for these two new features put them into different files (e.g., null_sql_test.cpp & in_sql_test.cpp)? Make sure you update the headers too.